### ISSUE TITLE
SelectOneRadio: HTML validation error when using required attribute

### ISSUE THREAD

**Issue by miedrl** [primefaces/primefaces#8324]
**Describe the defect**
When using selectOneRadio with attributes layout="grid", columns="3" and required="true", the selectOneRadio is rendered as a \<table>-element in the HTML DOM with the attribute aria-required="true". This HTML syntax causes a validation error when analyzing the site with the [Nu HTML Checker](https://validator.w3.org/nu/) as the attribute aria-required is not allowed on \<table>-elements.

**Environment:**
 - PF Version: _10.0_

**To Reproduce**
1. Add the example code below to your application.
2. Copy your site's HTML code into the [Nu HTML Checker](https://validator.w3.org/nu/) and start validation.
3. See validation error from Screenshot below.

**Expected behavior**
The attribute "aria-required" should not be added to the \<table>-element within selectOneRadios as it causes an HTML validation error and thus violates [WCAG success criterion 4.1.1](https://www.w3.org/TR/WCAG21/#parsing) for accessibility.

**Example XHTML**
```html
<h:form id="frmTest">
      <div class="p-grid ui-fluid">
         <div class="p-col-12">
            <div class="card">
               <h5>SelectOneRadio</h5>
               <p:selectOneRadio id="testRadio" columns="3" required="true" layout="grid">
                  <f:selectItem itemLabel="Option1" itemValue="Option1"/>
                  <f:selectItem itemLabel="Option2" itemValue="Option2"/>
                  <f:selectItem itemLabel="Option3" itemValue="Option3"/>
                  <f:selectItem itemLabel="Option4" itemValue="Option4"/>
                  <f:selectItem itemLabel="Option5" itemValue="Option5"/>
                  <f:selectItem itemLabel="Option6" itemValue="Option6"/>
                  <f:selectItem itemLabel="Option7" itemValue="Option7"/>
                  <f:selectItem itemLabel="Option8" itemValue="Option8"/>
                  <f:selectItem itemLabel="Option9" itemValue="Option9"/>
              </p:selectOneRadio>
            </div>
        </div>
      </div>
</h:form>
```
**Screenshot rendered HTML**
![Screenshot_HTML_SelectOneRadio](https://user-images.githubusercontent.com/77431692/150933107-ac86e016-7ea5-4fce-b175-807dc49ec5f6.png)

**Screenshot HTML validation error**
![Screenshot_HTML_Validation_SelectOneRadio](https://user-images.githubusercontent.com/77431692/150933165-3821d656-6c91-4538-8167-b0b1d603abea.png)

**Comment by jepsar** [comment]
This might already be fixed by #8194 where the role of the table is changed.

**Comment by miedrl** [comment]
@jepsar Thank you. I tested validation after changing the table's role to "radiogroup" and there is no more validation error. In which version of PrimeFaces will this fix be available?

**Comment by jepsar** [comment]
The PR is still open for review, but the issue is labeled 12.0.0.
